app/src/main/java/chat/rocket/android/members/adapter/MembersAdapter.kt

```diff
@@ -44,6 +44,8 @@ class MembersAdapter(private val listener: (MemberUiModel) -> Unit) :
             image_avatar.setImageURI(memberUiModel.avatarUri)
             text_member.content = memberUiModel.displayName
             setOnClickListener { listener(memberUiModel) }
+            val statusDrawable = DrawableHelper.getUserStatusDrawable(memberUiModel.status, context)
```

Comment:
Could you change to the below code, please?

```
            image_avatar.setImageURI(memberUiModel.avatarUri)
            text_member.content = memberUiModel.displayName
            image_member_status.setImageDrawable(DrawableHelper.getUserStatusDrawable(memberUiModel.status, context))
            setOnClickListener { listener(memberUiModel) }
```

Reply:
Changed it a bit differently to work with the other suggestion.